server/db/seeders/feedBackSampleData.js

```diff
@@ -0,0 +1,602 @@
+    "reviewId": 66,
+    "userId": 66,
+    "isHelpful": 0
+  },
+  {
+    "id": 67,
+    "reviewId": 67,
+    "userId": 67,
+    "isHelpful": 1
+  },
+  {
+    "id": 68,
+    "reviewId": 68,
+    "userId": 68,
+    "isHelpful": 1
+  },
+  {
+    "id": 69,
+    "reviewId": 69,
+    "userId": 69,
+    "isHelpful": 0
+  },
+  {
+    "id": 70,
+    "reviewId": 70,
+    "userId": 70,
+    "isHelpful": 1
+  },
+  {
+    "id": 71,
+    "reviewId": 71,
+    "userId": 71,
+    "isHelpful": 0
+  },
+  {
+    "id": 72,
+    "reviewId": 72,
+    "userId": 72,
+    "isHelpful": 1
+  },
+  {
+    "id": 73,
+    "reviewId": 73,
+    "userId": 73,
+    "isHelpful": 1
+  },
+  {
+    "id": 74,
+    "reviewId": 74,
+    "userId": 74,
+    "isHelpful": 1
+  },
+  {
+    "id": 75,
+    "reviewId": 75,
+    "userId": 75,
+    "isHelpful": 0
+  },
+  {
+    "id": 76,
+    "reviewId": 76,
+    "userId": 76,
+    "isHelpful": 1
+  },
+  {
+    "id": 77,
+    "reviewId": 77,
+    "userId": 77,
+    "isHelpful": 1
+  },
+  {
+    "id": 78,
+    "reviewId": 78,
+    "userId": 78,
+    "isHelpful": 0
+  },
+  {
+    "id": 79,
+    "reviewId": 79,
+    "userId": 79,
+    "isHelpful": 1
+  },
+  {
+    "id": 80,
+    "reviewId": 80,
+    "userId": 80,
+    "isHelpful": 1
+  },
+  {
+    "id": 81,
+    "reviewId": 81,
+    "userId": 81,
+    "isHelpful": 1
+  },
+  {
+    "id": 82,
+    "reviewId": 82,
+    "userId": 82,
+    "isHelpful": 1
+  },
+  {
+    "id": 83,
+    "reviewId": 83,
+    "userId": 83,
+    "isHelpful": 0
+  },
+  {
+    "id": 84,
+    "reviewId": 84,
+    "userId": 84,
+    "isHelpful": 1
+  },
+  {
+    "id": 85,
+    "reviewId": 85,
+    "userId": 85,
+    "isHelpful": 1
+  },
+  {
+    "id": 86,
+    "reviewId": 86,
+    "userId": 86,
+    "isHelpful": 1
+  },
+  {
+    "id": 87,
+    "reviewId": 87,
+    "userId": 87,
+    "isHelpful": 0
+  },
+  {
+    "id": 88,
+    "reviewId": 88,
+    "userId": 88,
+    "isHelpful": 0
+  },
+  {
+    "id": 89,
+    "reviewId": 89,
+    "userId": 89,
+    "isHelpful": 1
+  },
+  {
+    "id": 90,
+    "reviewId": 90,
+    "userId": 90,
+    "isHelpful": 1
+  },
+  {
+    "id": 91,
+    "reviewId": 91,
+    "userId": 91,
+    "isHelpful": 1
+  },
+  {
+    "id": 92,
+    "reviewId": 92,
+    "userId": 92,
+    "isHelpful": 1
+  },
+  {
+    "id": 93,
+    "reviewId": 93,
+    "userId": 93,
+    "isHelpful": 1
+  },
+  {
+    "id": 94,
+    "reviewId": 94,
+    "userId": 94,
+    "isHelpful": 1
+  },
+  {
+    "id": 95,
+    "reviewId": 95,
+    "userId": 95,
+    "isHelpful": 1
+  },
+  {
+    "id": 96,
+    "reviewId": 96,
+    "userId": 96,
+    "isHelpful": 1
+  },
+  {
+    "id": 97,
+    "reviewId": 97,
+    "userId": 97,
+    "isHelpful": 1
+  },
+  {
+    "id": 98,
+    "reviewId": 98,
+    "userId": 98,
+    "isHelpful": 0
+  },
+  {
+    "id": 99,
+    "reviewId": 99,
+    "userId": 99,
+    "isHelpful": 0
+  },
+  {
+    "id": 100,
+    "reviewId": 100,
+    "userId": 100,
+    "isHelpful": 1
+  }
+];
```

Comment:
It is possible to generate these values using faker but its little tricky at this point since my schema has many foreign_key dependency. My fetch query has inner joins and if a foreign key fails to refers primary key it will throw error. I just wants to have my db clean at this point so i can work with some data in the UI. I do have some plans to generate data dynamically with faker. Will be implemented that as a separate ticket.